src/components/login-signup/login-signup.jsx

```diff
@@ -2,37 +2,46 @@ import React, { useState } from 'react';
 import './login-signup.css';
 
 function Login() {
+    const [firstName, setFirstName] = useState('');
+    const [lastName, setLastName] = useState('');
     const [email, setEmail] = useState('');
+    const [username, setUserName] = useState('');
     const [password, setPassword] = useState('');
-    const [businessName, setBusinessName] = useState('');
-    const [isSignUp, setIsSignUp] = useState(false); // State to toggle between login and signup
+    const [isSignUp, setIsSignUp] = useState(false); // State to check if we're currently in login or signup
     const [error, setError] = useState('');
 
     const handleSubmit = async (e) => {
-        e.preventDefault();
-        const url = isSignUp ? `${import.meta.env.VITE_BACKEND_ADDRESS}/auth/signup` : `${import.meta.env.VITE_BACKEND_ADDRESS}/auth/login`;
-        const body = isSignUp ? JSON.stringify({ email, password, businessName }) : JSON.stringify({ email, password });
-
-        try {
-            const response = await fetch(url, {
-                method: 'POST',
-                headers: {
-                    'Content-Type': 'application/json'
-                },
-                body: body
-            });
-
-            const data = await response.json();
-            if (response.ok) {
-                localStorage.setItem('token', data.token);
-                // Redirect or update state to indicate success
-            } else {
-                setError(data.message);
-            }
-        } catch (error) {
-            setError('Operation failed. Please try again.');
-        }
-    };
+      e.preventDefault();
+      const url = isSignUp ? `${import.meta.env.VITE_BACKEND_ADDRESS}/auth/signup` : `${import.meta.env.VITE_BACKEND_ADDRESS}/auth/login`;
+      const body = isSignUp ? JSON.stringify({ email, password, businessName }) : JSON.stringify({ email, password });
+      if (email === '') {
+        setError('Please enter an email address');
+        return;
+    }
+      // Check if the email address ends with ".edu"
+      if (!email.endsWith('.edu')) {
+          setError('This is not a valid college email address');
+          return;
+      }
```

Comment:
Do all collage emails have .edu in them?

Reply:
Yes all colleges to my knowledge have the .edu suffix at the end of them. checking using this was a suggestion by Daniel, as opposed to using a school email verification API